source/docs/appendices/migration-guides.md

```diff
@@ -9,13 +9,22 @@ Our goal is to make migrations as smooth as possible. This is why we try to make
 
 ## `2.8.0` -> `2.9.0`
 
-No action needed for migrating to this version.
+### Customs options `price` and `price_type` field deprecations
+
+As of the version 2.9.0, the `price` and `price_type` fields of the types `ProductCustomOption` and `ProductCustomOptionValue` have been deprecated in order to provide both the price including and excluding taxes for each options. Custom options prices or rate are now carried by the `extraCost` field available in those types. To stop using those deprecated fields, the following files have been modified:
+
+- `theme/modules/Cart/CartItem/CartItemOptionsUpdater/CartItemOptionsUpdaterFragment.gql`
+- `theme/components/organisms/Configurator/ProductConfiguratorFragment.gql`
+- `theme/components/organisms/Configurator/CustomOption/CustomOptionLabelWithPrice.js`
+- `theme/modules/ProductView/withSelectedCustomOptions.js`
+
+So if you have overridden those files, you have to apply [the same update than the one done in Front-Commerce](https://gitlab.com/front-commerce/front-commerce/-/commit/d3dbac21907cb2eae21cef7876bcbdd7b642e66f?merge_request_iid=622).
```

Comment:
self note: update the commit hash before after the review and before merging this PR